components/src/jsMain/kotlin/dev/fritz2/components/validation/validation.kt

```diff
@@ -0,0 +1,38 @@
+package dev.fritz2.components.validation
+
+import dev.fritz2.binding.RootStore
+import dev.fritz2.binding.Store
+import dev.fritz2.binding.SubStore
+import dev.fritz2.validation.Validator
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+
+interface Validatable<D, T> : Store<D> {
```

Comment:
I think the name Validatable might be misleading here, because the point is if it really gets validated in this store, not if it is possible to validate. What do you think about HasValidation or WithValidator or just Validating

Reply:
Ok, renamed to `WithValidator` like we had in our core code.